index.html

```diff
@@ -11,14 +11,15 @@
     <title>Document</title>
 </head> 
 <body> 
-    <div class="flex-container">
-        <header>
+    <!--toolbar--> 
+        <header class="navigation">
             <div class="menu">
                 <a href=""> My Logo</a>
                 <div class="humberger">
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                    <span class="bar"></span>
+                    <nav class="bar"></nav>
+                    <nav class="bar"></nav>
+                    <nav class="bar"></nav>
+                    
                 </div>
```

Comment:
[OPTIONAL] ad requested by the previous reviewer this section shoul be like this :point_down: , as this is semantically more appropriate
```
<nav class="humberger">
            <div class="bar"></nav>
            <div class="bar"></nav>
             <div class="bar"></nav>
</nav>
```